pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -241,6 +242,14 @@ def set_certified(self, request, *args, **kwargs):
         return Response({})
 
 
+class CollectionVersionDocsViewSet(CollectionVersionViewSet):
```

Comment:
Is this viewset expected to support list() operation?

Reply:
no, just retrieve()... the urls.py only has the get()/retrieve()

Reply:
Sorry I missed this reply until now. Then I think we need to have this be its own viewset because by inheriting from `CollectionVersionViewSet` it also receives other action endpoints like `list` for example.

If their are common parts, then those could be moved to a mixin and shared between both classes.

Reply:
Updated per above.